src/models/PolicyList.ts

```diff
@@ -63,13 +63,27 @@ declare interface PolicyList {
 /**
  * The PolicyList caches all of the rules that are active in a policy room so Mjolnir can refer to when applying bans etc.
  * This cannot be used to update events in the modeled room, it is a readonly model of the policy room.
+ *
+ * You can inform the PolicyList about new events in a room with the `updateForEvent`, this will eventually
+ * cause the PolicyList to synchronize itself with the room if it doesn't know about that state event.
+ * Each time the PolicyList has finished synchronizing, it will emit the `'PolicyList.update'` event on itself as an EventEmitter.
```

Comment:
what is it synchronising? its view of its own room?

Reply:
yeah i'll update this doc to say which method is doing that